src/extension.ts

```diff
@@ -130,8 +131,15 @@ function displayLog(outputPanel: vscode.OutputChannel, message: string, show: bo
 }
 
 function executeCommand(command: (...args: any[]) => Promise<any>, thisArg: any, ...params: any[]) {
+    const errorMessage = params[params.length - 1];
```

Comment:
how can we be sure that the last entry in this array is a string message? 
To me this looks fragile and error-prone (what if a latter developer who's new to the project would pass in some object? Can't we use types to avoid such errors?

Reply:
I pass the string to this method voluntarily inside the registerCommands, so right now it will work without any check but I can add it easily.